test-case-reporting/src/setup_db.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+require('dotenv').config();
+
+import {Database, dbConnect} from './db';
+
+/** Set up the database table. */
+export async function setupDb(db: Database): Promise<unknown> {
+  return db.schema
+    .createTable('builds', table => {
+      table.increments('id').primary();
+      table.string('commit_hash', 40);
+      table.integer('pr_number');
+      table.dateTime('started_at');
+    })
+    .createTable('jobs', table => {
+      table.increments('id').primary();
+      table.integer('build_id').unsigned().notNullable();
+      table.string('job_number');
+      table.enu('type', ['unit', 'integration'], {
```

Comment:
1. it's better to leave this as a string - e.g., when we decide to add `e2e` tests, we could just change the code, and not have to make any (potentially dangerous) changes to the database!
Sometimes enums do make sense, when you know that the set of options is final and/or when type strictness is too important to leave it to strings :)
2. I think calling it `test_suite` or `test_suite_type` is cleaner but @rcebulko should have the final word on this one

Reply:
+1 `test_suite_type`

Reply:
Done.